kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/DataType.java

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+ * Copyright (c) 2016, 2024 Eurotech and/or its affiliates and others
+ * 
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ * 
+ * Contributors:
+ *  Eurotech
+ *  Amit Kumar Mondal
+ ******************************************************************************/
+
+/**
+ * This is a copy of {@code org.eclipse.kura.type.DataType}
+ */
+
```

Comment:
Why you needed to copy this class?

Reply:
Because i need them in client part of webui